Makefile

```diff
@@ -33,7 +33,7 @@ SUCCESS := $(BOLD)$(GREEN)
 
 # Test variables #################################
 # the quality gate lower threshold for unit test total % coverage (by function statements)
-COVERAGE_THRESHOLD := 45
+COVERAGE_THRESHOLD := 30
```

Comment:
go 1.22 changed the rules around how coverage is counted -- this new value reflects within 2% of where we are today